res/controllers/Numark_DJ2GO2_Touch_scripts.js

```diff
@@ -94,7 +94,8 @@ DJ2GO2Touch.Deck = function(deckNumbers, midiChannel) {
     this.tempoFader = new components.Pot({
         group: "[Channel" + script.deckFromGroup(this.currentDeck) + "]",
         midi: [0xB0 + midiChannel, 0x09],
-        key: "rate"
+        key: "rate",
+        invert: true
```

Comment:
DJ2GO2/DJ2GO2 Touch Tempo Fader sends MIDI Message,
Deck1 0xB0 0x09 value
Deck2 0xB1 0x09 value
value:
 Top (Back-side) 0x00
 Centre 0x40
 Bottom (Front-side) 0x7F
This will be inverted from many other controllers.

Reply:
Makes sense. I encounter similar behavior on an unmapped Numark controller that I have laying around. I wonder why this was not discovered by @fjordstrom